Sources/Sentry/SentryTracer.m

```diff
@@ -641,6 +645,10 @@ - (SentryTransaction *)toTransaction
 
 #if SENTRY_HAS_UIKIT
     [self addMeasurements:transaction];
+
+    if ([viewNames count] > 0) {
```

Comment:
`m`: Do we have tests for this new logic?

Reply:
Yeap!!